docs_src/microservices/core/metadata/Ch-Metadata.md

```diff
@@ -345,20 +345,25 @@ Sequence diagrams for some of the more critical or complex events regarding meta
 
 Please refer to the general [Common Configuration documentation](../../configuration/CommonConfiguration.md) for configuration properties common to all services. Below are only the additional settings and sections that are not common to all EdgeX Services.
 
+=== "Writable.ProfileChange"
+    |Property|Default Value|Description|
+    |---|---|---|
+    |StrictDeviceProfileChanges|false|Whether to allow device profile modifications, set to `true` to reject all modifications.|
```

Comment:
Whether to allow device profile modifications, set to `true` to reject all modifications which might impact the existing events and readings. Thus, the changes like `manufacture`, `isHidden`, or `description` can still be made.